inc/Engine/WPRocketUninstall.php

```diff
@@ -87,6 +87,23 @@ class WPRocketUninstall {
 		'used-css',
 	];
 
+	/**
+	 * WP Rocket Post MetaData Entries
+	 *
+	 * @var array
+	 */
+	private $wpr_post_meta = [
+		'_rocket_exclude_minify_css',
+		'_rocket_exclude_minify_js',
+		'_rocket_exclude_cdn',
+		'_rocket_exclude_lazyload',
+		'_rocket_exclude_lazyload_iframes',
+		'_rocket_exclude_async_css',
+		'_rocket_exclude_defer_all_js',
+		'_rocket_exclude_delay_js',
+		'_rocket_exclude_remove_unused_css',
+	];
```

Comment:
```suggestion
	/**
	 * WP Rocket Post meta data entries
	 *
	 * @var array
	 */
	private $post_meta = [
		'minify_css',
		'minify_js',
		'cdn',
		'lazyload',
		'lazyload_iframes',
		'async_css',
		'defer_all_js',
		'delay_js',
		'remove_unused_css',
	];
```

We can use the prefix in the delete function directly, it makes the array easier to read. We also don't need to prefix the private variable.